src/elements/tooltip.svelte

```diff
@@ -24,7 +23,7 @@ $: {
 let x = 0;
 let y = 0;
 
-export const recalculateStyle = async () => {
+export let recalculateStyle = async () => {
```

Comment:
Why the change to `let`?

Reply:
Sadly, using `const` produces a runtime error on the `create_custom_element` call:

```
TypeError: Cannot redefine property: recalculateStyle
```

Reply:
ahhh interesting